utils/nctl/sh/scenarios/gov96.sh

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env bash
+
+source "$NCTL"/sh/scenarios/common/itst.sh
+
+# Exit if any of the commands fail.
+set -e
+
+#######################################
+# Runs an integration tests that tries to simulate
+# stalling consensus by stopping all but one node. It
+# then restarts the nodes and checks for the chain
+# to progress.
+#
+# Test requested by one of the clients who wants to keep network operational
+# (answering queries) but not producing new blocks and resume it weeks later.
+#
+# Arguments:
+#   `timeout=XXX` timeout (in seconds) when syncing.
+#######################################
+function main() {
+    log "------------------------------------------------------------"
+    log "Starting Scenario: gov96"
+    log "------------------------------------------------------------"
+
+    # 0. Wait for network start up
+    do_await_genesis_era_to_complete
+    # 1. Allow chain to progress
+    do_await_era_change
+    # 2. Verify all nodes are in sync
+    check_network_sync
+    # 3-5. Stop three nodes
```

Comment:
```suggestion
    # 3-5. Stop four nodes
```
…and probably "4 - 6", changing all subsequent numbers?

Reply:
Fixed in [`070855e` (#1853)](https://github.com/casper-network/casper-node/pull/1853/commits/070855e155d41077944491d7afe5debdc3aace4f)